src/app/main-api-handler.ts

```diff
@@ -208,6 +218,23 @@ ipcMain.on(
         if (typeof arg.keyCode === 'number') {
           handleKeyPress(arg.keyCode);
         }
+        break;
+      case apiCmds.getMyPresence:
+        presenceStatus.setMyPresence(arg.status);
+        mainWebContents?.send('sys-tray-presence-update', {
```

Comment:
I would like to keep this whole thing here, since the API was sent from mana ssf and it is in the rightplace I guess (Following previously implnted architecture)

Reply:
So moving the mainwebcontent into setMyPresence is the best choice